src/site/navigation/facility_no_drupal_page_sidebar_nav.drupal.liquid

```diff
@@ -5,110 +5,114 @@
 
     For an example of how to make a string look like this, look at in `src/site/layouts/person_profile.drupal.liquid` where sidebarData is being assigned.
 {% endcomment %}
-<nav id="va-detailpage-sidebar" data-drupal-sidebar class="va-c-facility-sidebar usa-width-one-fourth va-sidebarnav">
-    <div>
-        <button class="va-btn-close-icon va-sidebarnav-close" type="button" aria-label="Close this menu"></button>
 
-        {% assign deepLinksObj = sidebarData | jsonToObj %}
-        <div class="left-side-nav-title">
-            <h4>{{ deepLinksObj.link.label }}</h4>
-        </div>
+{% if buildtype != 'vagovprod' %}
+    <script type="text/javascript">
+      window.sideNav = {{ sidebarData | json }};
+    </script>
 
-        {% assign depth = deepLinksObj.depth  %}
-        {% assign deepLinks = deepLinksObj.link.links %}
+    {% comment %} React Widget located in `src/platform/site-wide/side-nav` {% endcomment %}
+    <div data-widget-type="side-nav"></div>
```

Comment:
Couldn't seem to get the data property working with Liquid... If we do find out how to put stringified JSON as a data property in Liquid, I think it would be good to create a separate ticket and update all instances of where we are using globals vs using data properties for the React widgets! :)